src/views/UserEdit/UserEditHelpers.js

```diff
@@ -29,15 +29,19 @@ export function resourcesLoaded(obj, exceptions = []) {
  * @param {function} sendCallout function reference, likely this.context.sendCallout
  */
 export function showErrorCallout(res, sendCallout) {
+  console.error(res); // eslint-disable-line no-console
   res.text()
     .then(body => {
+      const isUserExists = body?.includes('username already exists');
```

Comment:
what is response type, is it always text? or json for caae when username is not unique?

Reply:
It is both. On **create** it **_json_** type and on **edit** it is **_text_** type.

Reply:
could you post both responses of POST/PUT uniqueness error response?

Reply:
![Screenshot 2023-08-08 at 12 54 26 PM](https://github.com/folio-org/ui-users/assets/116072773/ca8616e3-eef7-432b-8643-3bc754a0b29f)

![Screenshot 2023-08-08 at 12 53 15 PM](https://github.com/folio-org/ui-users/assets/116072773/0043149f-ad0a-4f59-b05b-efcc2fa61e60)

Reply:
I have updated the error handler function

Reply:
interesting, ok